fabric-registry-sync-v0/src/main/java/net/fabricmc/fabric/impl/registry/sync/FabricRegistryInit.java

```diff
@@ -0,0 +1,23 @@
+package net.fabricmc.fabric.impl.registry.sync;
+
+import net.fabricmc.api.ModInitializer;
+import net.fabricmc.fabric.api.event.registry.RegistryAttribute;
+import net.fabricmc.fabric.api.event.registry.RegistryAttributeRegistry;
+import net.minecraft.util.registry.Registry;
+
+public class FabricRegistryInit implements ModInitializer {
+	@Override
+	public void onInitialize() {
+		//TODO this is a basic list of what should be done, a full list needs to be created
```

Comment:
Maybe throw a comment in, like a `TODO` saying we need to add new registries to this list if a new one appears during an update.

Also could there be a way we can `detect` if a registry needs to be synced or persisted without having to manually specify these for all 39+ registries.

Reply:
Yes, this TODO here is just saying that its incomplete and need fully populating before its ready to be merged.

Im not sure of a way to be able to detect this, it would be nice to be able to ensure that we cover all bases. This is honesly going to be a big pita to keep up to date and correct, but I dont see another way to go about this.